bundles/ilg.gnumcueclipse.debug.gdbjtag.jumper/src/ilg/gnumcueclipse/debug/gdbjtag/jumper/preferences/DefaultPreferences.java

```diff
@@ -21,20 +21,18 @@
 
 	public static final boolean SERVER_DO_START_DEFAULT = true;
 	public static final boolean DO_START_GDB_SERVER_DEFAULT = true;
-	public static final String SERVER_EXECUTABLE_DEFAULT = "${qemu_path}/${qemu_executable}";
+	public static final String SERVER_EXECUTABLE_DEFAULT = "jumper";
```

Comment:
I see you define the variables to hold the path & executable name, why not use them?

Reply:
Oops! I saw you removed the preferences/properties pages to set the path & name. Are you sure you want to have everything hardcoded in the plug-in?

Reply:
i ended up removing that as we don't need that here. Jumper is always installed on the same location. I also removed the preferences and workspace setup pages.

Reply:
Maybe you discovered the secret of writing always upward compatible software. ;-)

I personally do not think this is realistic. At a certain point you'll have to break something in order to be able to further inovate. Once you do this, you'll need to bind a certain user application with a certain version of jumper. The consequence is that you'll have to be able to install multiple versions at the same time. Once you do this, you need a mechanism to handle multiple paths.

The current Linux paradigm to install a single version and to use the system folders, is anyway outdated, and should be used with caution.

The binary xPacks were invented exactly to overcome this limitation, you can install any set of versions for the same tool in parallel, and the install location is in the user home. Each client xPack list the exact dependencies, with versions, and will always use the desired version. 

Strict version control even for tools is also an absolute must for CI tasks.

Reply:
The first step is to update the procedure that builds the distribution binaries to create a subfolder that includes the full version number. Semver strongly recommended, since this greatly helps automated tools. For historical reasons (Eclipse bad influence) my version numbers also include the release date.

Then find a place in the user home folder. A good inspiration can be the folders used by my binaries (like `$HOME/opt/gnu-mcu-eclipse/qemu/2.8.0-3-20180521-1416` for POSIX, and somewhere in `AppData\Roaming\GNU MCU Eclipse\QEMU\2.8.0-3-20180521-1416` for Windows).

Reply:
Added the configuration menus back and the usage of the environment variable

Reply:
We expect users to separately install the Jumper tool. It's an easy pip install jumper operation. We also provide an install script on our site. So the jumper bin would not be delivered as a part of GNU arm eclipse. That's fine, right?

Reply:
I don't think it's a good idea to depend on a "global system" install of a tool like this.
Users might have good reason to have different versions installed and in different places not necessarily on the system path.
I think that the path should be configurable/parameterised as is the case with many other tools referenced by the GME plugins.
I think that this is basically what Liviu advised earlier too.
Hope this helps.

Reply:
yes that's perfect. I added the configurable parameters in the same way the QEMU plugin works.